### PR TITLE
Product detail page

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Made some HTML Semantic changes of putting main tag after header. Removed Hover effect btn of add to cart and placed a fixed btn on product card.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,6 +239,7 @@
                     <img src="https://rb.gy/w5n8ml" alt="featuring home decor product">
                 </div>
                 </section>
+                <main></main>
```
Comment (venkateshwarans) [inline index.html]: this is probably an oversight.?

File: index.html [hunk 2]
```diff
@@ -105,6 +104,7 @@
             </section>
         </aside>
 
+        <main>
```
Comment (venkateshwarans) [inline index.html:110]: I think this tag is not closed

File: index.html (hunk 1 above)
Comment (ShekharDhangar) [reply index.html]: Thanks for pointing, I'll change it

**Review by venkateshwarans — APPROVED** [review]
Added a comment. Apart from the rest looks fine to me.

File: index.html (hunk 2 above)
Comment (ShekharDhangar) [reply index.html:110]: No it is the opening tag

**Review by vrinda-mahajan — APPROVED** [review]
Looks good to merge.

**Pull request merged**